### PR TITLE
[operator-trivy] Add FAQ for module doc

### PR DESCRIPTION
**Pull request by alex123012**
## Description

This PR adds a FAQ for the trivy operator module

## Why do we need it, and what problem does it solve?
It is not clear from the CIS grafana dashboard how to view resources that have not passed CIS compliance checks

## What is the expected result?

Added FAQ to doc
## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: operator-trivy
type: chore
summary: Add a FAQ.
impact_level: low
```

### CODE REVIEW

**Review by nabokihms — DISMISSED** [review]

**Comment by nabokihms** [comment]
@z9r5 you can merge it yourself when ready.

File: ee/modules/500-operator-trivy/docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+---
+{% raw %}
+
+## How to view all resources that have not passed CIS compliance checks?
+
+```bash
+kubectl get clustercompliancereports.aquasecurity.github.io cis -ojson | jq '.status.detailReport.results | map(select(.checks | map(.success) | all | not))'
+```
+
+## How to view resources that have not passed specific CIS compliance checks
```
Comment (z9r5) [inline ee/modules/500-operator-trivy/docs/FAQ.md:12]:
```suggestion
## How to view resources that have not passed specific CIS compliance checks?
```

File: ee/modules/500-operator-trivy/docs/FAQ_RU.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Модуль operator-trivy: FAQ"
+---
+{% raw %}
+
+## Как посмотреть все ресурсы, которые не прошли  CIS compliance проверки?
```
Comment (z9r5) [inline ee/modules/500-operator-trivy/docs/FAQ_RU.md:6]:
```suggestion
## Как посмотреть все ресурсы, которые не прошли CIS compliance проверки?
```

File: ee/modules/500-operator-trivy/docs/FAQ_RU.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+```bash
+kubectl get clustercompliancereports.aquasecurity.github.io cis -ojson | jq '.status.detailReport.results | map(select(.checks | map(.success) | all | not))'
+```
+
+## Как посмотреть ресуры, которые не прошли конкретные CIS compliance проверки
```
Comment (z9r5) [inline ee/modules/500-operator-trivy/docs/FAQ_RU.md:12]:
```suggestion
## Как посмотреть ресуры, которые не прошли конкретную CIS compliance-проверку?
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Review by z9r5 — APPROVED** [review]

**Pull request merged**